webview.h

```diff
@@ -142,9 +157,12 @@ WEBVIEW_API void webview_return(webview_t w, const char *seq, int status,
 #include <cstring>
 
 namespace webview {
+
 using dispatch_fn_t = std::function<void()>;
```

Comment:
Note that `dispatch_fn_t` (`std::function<void()>`) is not in the `detail` namespace.

`win32_edge_engine::dispatch` takes `dispatch_fn_t` as an argument; however, `cocoa_wkwebview_engine::dispatch` and `gtk_webkit_engine::dispatch` take `std::function<void()>` which made me a bit confused.

Since the `webview` class inherits from one of the engine classes with `public` specified, `webview` exposes some details from the engine classes to users of the`webview` class. This means that `dispatch_fn_t` is also exposed and therefore part of the public C++ API.

Any comments?

Reply:
I was thinking about that earlier, and I came to the same conclusion. I agree that `dispatch_fn_t` should be under the `webview` namespace.